cmd/entrypoint/testutil_fake_syntheticauth_test.go

```diff
@@ -0,0 +1,119 @@
+package entrypoint_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/datawire/ambassador/v2/cmd/entrypoint"
+	v3bootstrap "github.com/datawire/ambassador/v2/pkg/api/envoy/config/bootstrap/v3"
+	v3cluster "github.com/datawire/ambassador/v2/pkg/api/envoy/config/cluster/v3"
+	"github.com/datawire/ambassador/v2/pkg/snapshot/v1"
+)
+
+func HasAuthService(namespace, name string) func(snapshot *snapshot.Snapshot) bool {
```

Comment:
The way the Fake framework uses predicates to find snapshots and Envoy configs makes it pretty much _always_ worth talking about what your predicates do, where they'll be used, etc.